src/main/scala/org/zalando/spearheads/innkeeper/Routes.scala

```diff
@@ -25,8 +25,9 @@ class Routes @Inject() (
     postRoutes: PostRoutes,
     getUpdatedRoutes: GetUpdatedRoutes,
     getDeletedRoutes: GetDeletedRoutes,
-    metrics: RouteMetrics,
-    authService: AuthService,
+    metrics: RouteMetrics)(
+    implicit
+    val authService: AuthService,
```

Comment:
can't do that :( that's how scalariform formats it... I tried to change the style, but it seems it's not possible... :-1: